src/chia_log/parsers/block_parser.py

```diff
@@ -0,0 +1,48 @@
+# std
+import re
+import logging
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List
+
+# lib
+from dateutil import parser as dateutil_parser
+
+
+@dataclass
+class BlockMessage:
+    """Parsed information from full node logs"""
+
+    timestamp: datetime
+    blocks_count: int
+
+class BlockParser:
+    """This class can parse info log messages from the chia farmer
+
+    You need to have enabled "log_level: INFO" in your chia config.yaml
+    The chia config.yaml is usually under ~/.chia/mainnet/config/config.yaml
+    """
+
+    def __init__(self):
+        logging.info("Enabled parser for blocks stats.")
+        # Doing some "smart" tricks with this expression to also match the 64th signage point
+        # with the same regex expression. See test examples to see how they differ.
+        self._regex = re.compile(
+            r"([0-9:.]*) full_node (?:src|chia).full_node.full_node\s*: INFO\s* (🍀\s* Farmed unfinished_block)"
```

Comment:
I would expect it doesn't and update the regex regardless. Makes it a little more difficult to read, but more robust in case they ever remove those emojis...